examples/booktest/mysql/models.go

```diff
@@ -26,6 +28,7 @@ type Author struct {
 
 type Book struct {
 	BookID        int
+	BookUUID      uuid.UUID
```

Comment:
The booktest examples is taken from [xo/xo](https://github.com/xo/xo/tree/master/examples/booktest). The PostgreSQL and MySQL versions should be the same as that project.

Can you back out your multiple changes to this file and add tests for overrides in the `internal/endtoend/testdata` directory?

Reply:
@kyleconroy How do you think we should organize the new `testdata` directory with multiple database engines?

Reply:
Individual tests should be in a subdirectory of testdata. Unless we're testing multi-engine configuration, we probably want to keep tests for each engine separate.